Library/Homebrew/diagnostic.rb

```diff
@@ -113,7 +113,7 @@ def examine_git_origin(repository_path, desired_origin)
         return if !Utils.git_available? || !repository_path.git?
 
         current_origin = repository_path.git_origin
-
+        user_origins = [HOMEBREW_BREW_REMOTE, HOMEBREW_CORE_REMOTE]
```

Comment:
Change https://github.com/Homebrew/brew/blob/e112110297be66f1ef6d396d80f2a7093643ef67/Library/Homebrew/diagnostic.rb#L563-L569 instead so that this is checked correctly for each repository.

Reply:
@hyuraku Still need this change, FYI.